src/services/ai-server/ai-persona-service/ai.persona.service.service.ts

```diff
@@ -175,4 +194,80 @@ export class AiPersonaServiceService {
       userID: aiPersonaService.id, // TODO: clearly wrong, just getting code to compile
     });
   }
+
+  private encryptExternalConfig(
```

Comment:
this is clear utility, split in a separate service

Reply:
Fixed.